src/napari_micromanager/main_window.py

```diff
@@ -344,6 +354,23 @@ def _get_defaultdict_layers(self, event: ActiveMDAEvent) -> defaultdict[Any, set
                 layergroups[key].add(lay)
         return layergroups
 
+    def _get_scale_from_sequence(
+        self, sequence: MDASequence, layer: napari.layers.Image
+    ) -> Tuple[float, ...]:  # noqa: U006
```

Comment:
i _think_ you should only need to use U006 when you're doing a module level type annotation... but all function parameters and return annotations shouldn't need them?  what happens if you remove?  (believe it will just cast it to `tuple[]` which should be fine here, since we have `from __future__ ... `